### PR TITLE
Create Empty Image Pull Secret when no default image pull secret specified.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Fixes: https://issues.redhat.com/browse/ACM-9759

### CODE REVIEW

**Comment by xuezhaojun** [comment]
/assign @zhiweiyin318 
/assign @qiujian16 
cc @hchenxa

File: pkg/bootstrap/imagepullsecret.go [hunk 1]
```diff
@@ -72,17 +74,27 @@ func getImagePullSecret(ctx context.Context, clientHolder *helpers.ClientHolder,
 
 func getDefaultImagePullSecret(ctx context.Context, clientHolder *helpers.ClientHolder) (*corev1.Secret, error) {
 	var err error
+	var secret *corev1.Secret
 
 	defaultSecretName := os.Getenv(constants.DefaultImagePullSecretEnvVarName)
 	if defaultSecretName == "" {
-		// Ignore the image pull secret, it can't be found from env DEFAULT_IMAGE_PULL_SECRET
-		return nil, nil
+		// If default secret can't be found from env DEFAULT_IMAGE_PULL_SECRET, create an empty image pull secret
```
Comment (zhujian7) [inline pkg/bootstrap/imagepullsecret.go:81]: nit: add more notes for why we need an empty pull secret.

**Pull request merged**